docs/guide/writing/docs-principles.rst

```diff
@@ -0,0 +1,367 @@
+Documentation Principles
+========================
+
+Software development benefits from `philosophies`_ and `principles`_ such as
+`DRY`_, `KISS`_, `code reuse`_, and many more. With these commonly understood
+and accepted standards, developers can hold themselves and each other
+accountable to producing high-quality code.
+
+.. _philosophies: https://en.wikipedia.org/wiki/Category:Software_development_philosophies
+.. _principles: https://en.wikipedia.org/wiki/Category:Programming_principles
+.. _DRY: https://en.wikipedia.org/wiki/Don%27t_repeat_yourself
+.. _Don't Repeat Yourself: https://en.wikipedia.org/wiki/Don%27t_repeat_yourself
+.. _KISS: https://en.wikipedia.org/wiki/KISS_principle
+.. _Code Reuse: https://en.wikipedia.org/wiki/Code_reuse
+
+This set of principles seeks to define similar standards for software
+*documentation* that, when practiced, will foster clean and intuitive
+content. The end goal is ultimately to delight and empower readers by
+making information easier for them to acquire.
+
+Overview, by component
+----------------------
+
+All documentation
+~~~~~~~~~~~~~~~~~
+
+In general, documentation should be...
+
+* `Precursory <#precursory>`__
+* `Participatory <#participatory>`__
+
+Content
+~~~~~~~
+
+*"Content" is the conceptual information within documentation.*
+
+**Content** should be...
+
+* `ARID <#arid>`__
+* `Skimmable <#skimmable>`__
+* `Exemplary <#exemplary>`__
+* `Consistent <#consistent>`__
+* `Current <#current>`__
+
+Sources
+~~~~~~~
+
+*A "source" refers to a system used to store and edit content.
+Examples of sources include: text files written using
+reStructuredText or Markdown, HTML content in a CMS database, help
+text stored within strings in application code, code comments to be
+assembled later into formalized documentation, and others too.*
+
+All **sources** should be...
+
+* `Nearby <#nearby>`__
+* `Unique <#unique>`__
+
+
+Publications
+~~~~~~~~~~~~
+
+*A "publication" refers to a single, cohesive tool that readers use to consume
```

Comment:
Agreed. I struggled with this word choice a bit, since I wanted it to encompass a wide array of "things" that the reader uses. Do you have a better word to suggest?